addons/website_forum/models/forum_post.py

```diff
@@ -23,7 +23,7 @@ class Post(models.Model):
         'website.seo.metadata',
         'website.searchable.mixin',
     ]
-    _order = "is_correct DESC, vote_count DESC, write_date DESC"
+    _order = "is_correct DESC, vote_count DESC, bump_date DESC"
```

Comment:
This is useful to match backend with frontend, right?

Reply:
Yes, unless we don't want to use bump date as default sorting for the backend.